Description: Correct pod NAME issue in LedgerSMB/DBObject/Date.pm
 Is missing the dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472559&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Date.pm
+++ b/LedgerSMB/DBObject/Date.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::Date Date Handling Back-end Routines for LedgerSMB
+LedgerSMB::Date - Date Handling Back-end Routines for LedgerSMB
 
 =head1 SYNOPSIS
 
